core/lib/config/src/configs/chain.rs

```diff
@@ -100,6 +100,9 @@ pub struct StateKeeperConfig {
     /// The maximum amount of pubdata that can be used by the batch. Note that if the calldata is used as pubdata, this variable should not exceed 128kb.
     pub max_pubdata_per_batch: u64,
 
+    /// The maximum number of DA slots that can be used, Calldata should only use 1. Blobs can use 1 or 2
+    pub max_number_da_slots: u64,
```

Comment:
A more suitable type may be used here: it is not likely to be 0 and it is not likely to go over 255 so maybe [`NonZeroU8`](https://doc.rust-lang.org/std/num/struct.NonZeroU8.html) would be bit more reasonable choice.

Reply:
i agree it can be a `u8`, the struct doesn't implement default though which has some downsides. we can always check the value once the config is read from `env`

Reply:
although i think we will remove all together since state keeper just cares about over size